### DISCUSSION TITLE
Strftime variables in the record filename are all converted to lowercase

### DISCUSSION THREAD

**Discussion by WasabiNME** [mrlt8/docker-wyze-bridge#487] [General]
The example in the docs:
`RECORD_FILE_NAME={cam_name}_%Y%m%d_%H_%M_%S`

Makes files named:
`mailbox_220724_Jul_07_1658702724.mp4`

The uppercase Y should produce a four-digit year instead of two digits, the uppercase M should produce minutes instead of the month, etc.

Is the implementation broken, or is it something to do with the mapping of the record folder to Windows, or me screwing something up (the most likely option)?

**Comment by mrlt8** [comment]
Can confirm, this is a bug. Pushing a fix now.
Thanks for letting me know!
